Makefile

```diff
@@ -93,6 +94,15 @@ configure-installer-tests-cluster:
 test:
 	go test -race $(PKGS)
 
+# Run clean test template spec parallelly
+.PHONY: clean-test
+clean-test:
+ifeq ($(SPEC_EXEC_METHOD),series)
+	go test -v github.com/openshift/odo/tests/template --ginkgo.focus="Example of a clean test" -ginkgo.slowSpecThreshold=$(SLOW_SPEC_THRESHOLD) -ginkgo.v -timeout $(TIMEOUT)
```

Comment:
I don't like that we use "Example of a clean test" it makes no sense. `clean-test` is running all e2e tests correct?

Reply:
No, ```clean-test`` is just a template for reference of how to write clean and efficient integration/e2e test.

Reply:
if it is just a template why do we need it in the makefile?

Reply:
It is just for a reference for @mohammedzee1000 and @dharmit who are also going to pick some of the test. In the very next sub task of #1473.

Anyway i will add a note there so that we don't forget to remove it in the next subtask fix.